doc/index.rst

```diff
@@ -125,10 +131,10 @@ stream of JSON objects on standard output. The ``baton`` programs are:
   Print the paths, metadata and access control lists of collections
   and data objects matching queries on metadata.
 
-* `baton-metasuper`
+* `baton-do`_
 
-  Replace all :term:`AVU` s in the metadata on collections and data
-  objects.
+  Perform a mixture of list, chmod, get, metamod or metaquery ``baton``
```

Comment:
I'm intending to remove baton-metasuper because everything it does can be done with the other tools.